### PR TITLE
Pluralize Fixture classes.

### PR DESCRIPTION
**Pull request by dereuromark**
Currently, the `public $fixtures = array('app.comment')` gets corrected to 'app.comments' etc.
But the class names itself not, throwing errors then.

This aims to correct the class names as well as the file names then of those classes.
It works, but not in a single step.
I have to run it twice to make both changes work. Any ideas why?

### CODE REVIEW

**Comment by lorenzo** [comment]
Maybe the problem is that the composer autoloader needs to be regenerated? Or that `class_exists()` lookups are cached in the process?

**Comment by dereuromark** [comment]
I think it has to do with the `$this->Stage->change()` and `$this->Stage->move()` calls.

File: src/Shell/Task/FixturesTask.php [hunk 1]
```diff
@@ -120,7 +121,35 @@ protected function _process($path) {
...
+				$pluralProcessor,
+				$contents,
+				-1,
+				$count
+			);
+			$newPath = dirname($path) . DS . $className . '.php';
+		}
+
+		$result = $this->Stage->change($path, $original, $contents);
+		$result &= $this->Stage->move($path, $newPath);
```
Comment (markstory) [inline src/Shell/Task/FixturesTask.php]: Why a reference?

Comment (dereuromark) [inline src/Shell/Task/FixturesTask.php]: Ops, that should have been a operator to make a combined boolean result from both commands.

Comment (dereuromark) [inline src/Shell/Task/FixturesTask.php]: Actually, that's not a reference in this case, but a bitwise operator.

The same as saying

    $result = true;
    $result = $result & true;
    return $result;

But I can rewrite that part.

Comment (markstory) [inline src/Shell/Task/FixturesTask.php]: It looks confusingly like a reference operator.

Comment (dereuromark) [inline src/Shell/Task/FixturesTask.php]: Yep. I can change that. But even written as the last line it does not work.
change() + move() together is broken right now.

**Pull request closed without merging**